public/js/monitor.js

```diff
@@ -1,45 +1,91 @@
 "use strict";
+/* global ga */
 
-function isValidEmail(val) {
-  // https://stackoverflow.com/a/46181
-  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-  return re.test(String(val).toLowerCase());
+function sendEvent(category, action, label){
+  ga("send", "event", category, action, label);
 }
 
-function enableBtnIfEmailValid(e) {
-  const emailBtn = document.getElementById("submit-email");
-  if (isValidEmail(e.target.value)) {
-    emailBtn.disabled = false;
-  } else {
-    emailBtn.disabled = true;
+function handleClicks(string){
+  const eventCategory = "Sign Ups";
+  const eventAction = "Clicked Sign Up";
+  if(document.getElementById("header-button")){
+    sendEvent(eventCategory, eventAction, "Sign up button clicked from featured breach.");
+  }
+  else if (document.getElementById("what-to-do")) {
+    sendEvent(eventCategory, eventAction, "Sign up button clicked after scanning email with breaches.");
+  }
+  else {
+    const headlineText = document.getElementsByTagName("h2")[0].innerHTML;
```

Comment:
This feels scary. Do you just want to give that “h2” a unique id or class instead?